src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -249,12 +249,12 @@
  * Now let's create a method handle for the comparator method defined above:
  *
  * {@snippet lang = java:
- * FunctionDescriptor comparDesc = FunctionDescriptor.of(JAVA_INT,
+ * FunctionDescriptor compareDesc = FunctionDescriptor.of(JAVA_INT,
```

Comment:
note - the use of `compar` here was deliberate, as it matched the name of the function pointer param in qsort - but I'll leave to you (e.g. in case that is too subtle)